src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      this.method = callback;
+
+      this.method(number);
```

Comment:
You don't need to store the method in the calculator object. You can directly call the bound callback. Replace 'this.method(number);' with 'callback.call(this, number);'